plugins/bcm57xx/README.md

```diff
@@ -26,3 +26,7 @@ Vendor ID Security
 ------------------
 
 The vendor ID is set from the PCI vendor, in this instance set to `PCI:0x14E4`
+
+External interface access
+---
+????
```

Comment:
the ethtool ioctl interface on the PCI device, and in the case of the recovery device raw writes to the PCI BARs